app/components/shared/review_component/review_component.html.slim

```diff
@@ -0,0 +1,13 @@
+.review-component
+  .review-component__heading
+    h2.govuk-heading-m.review-component__section#job_location_heading
```

Comment:
`#<step_name>_heading` needs to be dynamic (although I'd rather remove all the `new_attributes` code - but not sure we can)